install/installation.md

```diff
@@ -83,6 +83,13 @@ sudo python bootstrap_gunicorn.py
 * If `gunicorn` boots up successfully, most relevant logs will be stored in
  the `logs` directory. Otherwise they'll likely be in `syslog`.
 
+* If it shows the error regarding the `libvirt`, then there is missing `fftw3.h` file. Try the following:
+```
+sudo apt-get install libvirt-dev
```

Comment:
Can you give an instance when this error occurs? A screenshot would do.

Reply:
The problem with missing ```fftw3.h``` file occurs on Ubuntu like systems. Have no idea, why this is file is missing out of the box. On windows and CentOS this message does not appear.

About ```sudo apt-get install libvirt-dev```, most of the machines, doesn't have libvirt by default, so I was thinking that this is an essential step which is missed in the installation.

Reply:
This error appears on installation step

Reply:
Sounds good to me. I asked for the details so that if we ever encounter the same, we know where to start.

Reply:
Libvirt-bin is installed through the install.sh however, which should (in theory) be enough in combination with the python-libvirt package. I don't mind having the additional note in there though.